libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package com.trendyol.timelineview
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.trendyol.timelineview.databinding.ItemTimelineBinding
+
+class TimelineItemAdapter : RecyclerView.Adapter<TimelineItemAdapter.TimelineViewHolder>() {
+
+    private var timelineItems: MutableList<TimelineItemViewState> = mutableListOf()
+
+    @JvmOverloads
+    fun submitList(list: List<TimelineItemViewState> = emptyList()) {
```

Comment:
should be added notifyDataSetChanged

Reply:
not necessary

Reply:
We must add notifyDataSetChanged inorder to update list items when previous items are laid out.

Reply:
added